repository-setup.md

```diff
@@ -0,0 +1,27 @@
+# Repository Setup
+
+## Adding your tests to Prow
+
+For triggering and running tests from github, the Kubeflow org uses Prow, K8s' continuous integration tool.
+
+Follow this [pull request](https://github.com/kubernetes/test-infra/pull/7313/files) as a guide for setting up Prow.  Your pull request should:
```

Comment:
Can you link to these instructions
https://github.com/kubeflow/testing#adding-an-e2e-test-for-a-new-repository
instead of the pull request